controllers/storagecluster/storageconsumer.go

```diff
@@ -130,3 +108,132 @@ func (s *storageConsumer) ensureDeleted(_ *StorageClusterReconciler, _ *ocsv1.St
 	// cleaned up via owner references
 	return ctrl.Result{}, nil
 }
+
+func getStorageClassesForDistribution(ctx context.Context, kubeClient client.Client, storageCluster *ocsv1.StorageCluster) (
```

Comment:
There is no reason to mention distribution as the list coming our of this function is only used for local storage classes.  
```suggestion
func getLocalStorageClassNames(ctx context.Context, kubeClient client.Client, storageCluster *ocsv1.StorageCluster) (
```

Reply:
done.